integration/use-cases/operator-deployment.js

```diff
@@ -1,44 +1,57 @@
 const utils = require("./lib/utils");
-const {
-  screenshotsFolder,
-} = require("../args");
 const path = require("path");
 
 // The operator may take some minutes to be created
 jest.setTimeout(360000);
 
 test("Deploys an Operator", async () => {
+  // ODIC login
+  await page.goto(getUrl("/#/c/default/ns/kubeapps/operators"));
+  await page.waitForNavigation();
+  await expect(page).toClick("cds-button", { text: "Login via OIDC Provider" });
+  await page.waitForNavigation();
+  await expect(page).toClick(".dex-container button", { text: "Log in with Email" });
+  await page.waitForNavigation();
+  await page.type("input[id=\"login\"]", "kubeapps-operator@example.com");
+  await page.type("input[id=\"password\"]", "password");
+  await page.waitForSelector("#submit-login", { visible: true, timeout: 10000 });
+  await page.evaluate((selector) => document.querySelector(selector).click(), "#submit-login");
+  await page.waitForSelector(".kubeapps-header-content", { visible: true, timeout: 10000 });
   await page.goto(getUrl("/#/c/default/ns/kubeapps/operators"));
 
-  await expect(page).toFillForm("form", {
-    token: process.env.ADMIN_TOKEN,
-  });
-
-  await page.evaluate(() =>
-    document.querySelector("#login-submit-button").click()
-  );
+  // wait for the loading msg to disappear
+  await page.waitForFunction(() => !document.querySelector(".margin-t-xxl"));
 
   // Browse operator
   await expect(page).toClick("a", { text: "prometheus" });
 
-  await utils.retryAndRefresh(page, 2, async () => {
+  await utils.retryAndRefresh(page, 3, async () => {
     // Sometimes this fails with: TypeError: Cannot read property 'click' of null
     await expect(page).toClick("cds-button", { text: "Deploy" });
   });
 
-  // Deploy the Operator
-  await expect(page).toClick("cds-button", { text: "Deploy" });
+  const isAlreadyDeployed = await page.evaluate(() => document.querySelector('cds-button[disabled]') !== null);
```

Comment:
Exactly, I wanted to put a comment on this line for explaining so, indeed. 
In the CI environment, will never be already deployed, but in the local env it becomes easier to have this guard so that it doesn't matter if you already have operators installed.